hera_mc/cm_hookup.py

```diff
@@ -71,7 +116,9 @@ def get_hookup(self, hpn_list, rev, port_query, at_date,
                                                at_date=self.at_date,
                                                exact_match=exact_match,
                                                full_version=False)
-        hookup_dict = {'hookup': {}, 'fully_connected': {}, 'parts_epoch': []}
+        print(hpn_list)
+        print(parts)
```

Comment:
These look like debugging print statements that should not be kept in the final code.